src/app/groups/components/comments/CommentList.tsx

```diff
@@ -78,6 +84,24 @@ const CommentItem = ({
     setEditContent(comment.content);
   };
 
+  const handleReply = () => {
+    setIsReplyFormOpen(true);
+    setReplyContent('');
+  };
+
+  const handleReplySubmit = () => {
+    if (replyContent.trim()) {
+      onReply(comment.id, replyContent.trim());
+      setIsReplyFormOpen(false);
+      setReplyContent('');
+    }
+  };

```

Comment:
```suggestion
  const handleReplySubmit = () => {
    const trimmedReplyContent = replyContent.trim();
    const hasReplyConent = trimmedReplyContent.length > 0;

    if (hasReplyConent) {
      onReply(comment.id, trimmedReplyContent);
      setIsReplyFormOpen(false);
      setReplyContent('');
    }
  };
```

@yihimin  replyContent.trim() 를 2번 사용중에 있어서 변수로 빼면 중복 호출을 방지할 수 있을 것 같습니다.
또한, 좀 더 명확하게 변수를 정의해서 사용하면 좋을 것 같다는 생각입니다